docs/lab_08.html

```diff
@@ -98,12 +98,50 @@ <h2>Goals</h2>
-<p>So commit now and check the status.</p>
+<p>Previously, when committing the <code>hello.js</code> file, you entered the command <code>git commit -m &ldquo;Your commit message&rdquo;</code>. The <code>-m</code> flag will accept the commit message inline, within the &ldquo;double quotes&rdquo;. If you enter the command <code>git commit</code> without the <code>-m</code> flag, your default terminal editor will open </p>
+<p>If you omit the <code>-m</code> flag from the <code>git commit</code> command, git will open your default terminal editor.</p> 
+<p>The editor is chosen from the following list (in priority order):</p>
+<ul>
+  <li>GIT_EDITOR environment variable</li>
+	<li>core.editor configuration setting</li>
+	<li><span class="caps">VISUAL</span> environment variable</li>
+	<li><span class="caps">EDITOR</span> environment variable</li>
+</ul>
+<p>I have the <span class="caps">EDITOR</span> variable set to <code>vim</code>. Here's an example of what that would look like:</p>
+<pre class="sample">|
+  # Please enter the commit message for your changes. Lines starting
+  # with '#' will be ignored, and an empty message aborts the commit.
+  # On branch master
+  # Changes to be committed:
+  #   (use "git reset HEAD &lt;file&gt;..." to unstage)
+  #
+  #	modified:   hello.js
+  #
+</pre>
+<p>On the first line, you would enter the commit message for example: &#8220;Using process.argv&#8221;.  Save the file and exit the editor. </p>
+<p>Here are some basic commands for vim:</p>
+<ul>
+  <li>To add a commit message, you would type <code>i</code> to enter insert mode, then type your message.</li>
+  <li>To save and exit the file in vim, you would type <code>:wq</code> and press enter.  The <code>:</code> puts vim in command mode, the <code>w</code> writes the file, and the <code>q</code> quits the editor.</li>
+  <li>To exit without saving, you would type <code>:q!</code> and press enter.  The <code>!</code> forces the quit without saving.</li>
+</ul>
+<p>Once you have saved and exited the editor, you will see the commit message and the changes that were committed.</p>
+<pre class="sample">git commit
+  Waiting for Vim...
+  [master 569aa96] Using process.argv
+  1 files changed, 1 insertions(+), 1 deletions(-)
+</pre>
+<p>The &#8220;Waiting for Vim&#8230;&#8221; line comes from the <code>vim</code> program which sends the file to a running vim program and waits for the file to be closed.  The rest of the output is the standard commit messages.</p>
+
+<h2>Commit the change</h2>
+<p> For this assignment we will use the <code>-m</code> flag so that you can commit in the simulation terminal. So commit now and check the status.</p>
+
 <h3><b>Execute:</b></h3>
-<pre class="instructions">git commit -m "Using process.argv</pre>
+<pre class="instructions">git commit -m "Using process.argv"</pre>
 <pre class="instructions">git status</pre>
 </pre>
```

Comment:
I believe this is extra. completely breaks the formatting. definitely double check

```suggestion
```

Reply:
yes it was extra